packages/block-library/src/query/edit/query-inspector-controls.js

```diff
@@ -4,18 +4,30 @@
 import { PanelBody, QueryControls } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { InspectorControls } from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
 
 export default function QueryInspectorControls( { query, setQuery } ) {
-	const { order, orderBy } = query;
+	const { order, orderBy, author: selectedAuthorId } = query;
+	const { authorList } = useSelect( ( select ) => {
+		const { getEntityRecords } = select( 'core' );
+		return {
+			authorList: getEntityRecords( 'root', 'user', { per_page: -1 } ),
+		};
+	} );
 	return (
 		<InspectorControls>
-			<PanelBody title={ __( 'Sorting' ) }>
+			<PanelBody title={ __( 'Sorting and Filtering' ) }>
```

Comment:
```suggestion
			<PanelBody title={ __( 'Sorting and filtering' ) }>
```

Context: Adoption of "sentence case" in #18758